### PR TITLE
[GCCcore/6.4.0, GCCcore/7.3.0] Perl - Added XML::Parser XML::RegExp XML::DOM

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/f0fea133ef04fa41fa0315e6d8911eec for a full test report.

File: easybuild/easyconfigs/p/Perl/Perl-5.26.1-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -1447,6 +1447,21 @@ exts_list = [
         'source_urls': ['https://cpan.metacpan.org/authors/id/R/RJ/RJBS'],
         'checksums': ['67583075f6d62b78aaa956389f113df789ddd41911394092a5a2b8c6fa863070'],
     }),
+    ('XML::Parser', '2.44', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Perl/Perl-5.26.1-GCCcore-6.4.0.eb]: @vanzod This one is already there (as `2.44_01`) in this easyconfig

File: easybuild/easyconfigs/p/Perl/Perl-5.28.0-GCCcore-7.3.0.eb [hunk 2]
```diff
@@ -1442,6 +1442,21 @@ exts_list = [
         'source_urls': ['https://cpan.metacpan.org/authors/id/R/RJ/RJBS'],
         'checksums': ['67583075f6d62b78aaa956389f113df789ddd41911394092a5a2b8c6fa863070'],
     }),
+    ('XML::Parser', '2.44', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Perl/Perl-5.28.0-GCCcore-7.3.0.eb]: @vanzod Same here, already there (with newer version)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3134.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/3ef4913c46c2ab4d7da7e7355573464b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2594.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/134a1a86019c7d2774ce894abe3c393d for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**